nss-cohort-36/bangazon-client-imagination-station-react
# Lr remove comments 88

# Description
Removed all stray comments and console logs. Added styles to the ENTIRE APP. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
1. `git fetch --all`
2. `git checkout lr-remove-comments-88`
3. Walk through the app from start to finish and make sure EVERYTHING works, please.

# Checklist:
